packages/core/src/state-management/repository-internals.ts

```diff
@@ -294,6 +294,78 @@ export class RepositoryInternals {
     return this.processAnchorResponse(state$, anchorStatus$)
   }
 
+  /**
+   * Handle CASResponse and update state$.
+   *
+   * @param state$ - RunningState instance to update.
+   * @param asr - response from CAS.
+   * @return boolean - `true` if polling should stop, `false` if polling continues
+   */
+  async handleAnchorResponse(state$: RunningState, asr: CASResponse): Promise<boolean> {
```

Comment:
Why asr as the name for the cas response? Can we name it `casResponse` or just `resp`?

Reply:
gonna address that in a followup PR

Reply:
or, okay

Reply:
renamed